aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -87,6 +87,13 @@ Tensor isnan(const Tensor& self) {
   return self != self;
 }
 
+Tensor isfinite(const Tensor& self) {
+  if (!self.is_floating_point()) {
```

Comment:
Might be good to include part of the original comment, i.e., "Integral tensor types are finite"